advanced/creating-a-skin.md

```diff
@@ -88,76 +90,33 @@ Following is an example of a sample `skin.less` file:
 
 @import 'src/less/theme/theme';
 
+//
+// Place your variables here
 //
 
-// Root variables
-// Begin customization by changing these variables as most other variables are derivatives of these.
-@background-color: #1976D2;
-@base-value: 24px;
-@color-black: #222f3e;
-@color-tint: #3498db;
-@color-white: #fff;
-@color-error: #c00;
-@font-stack: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
-
-// Colors
-@border-color: darken(@background-color, 20);
-@text-color: contrast(@background-color, @color-black, @color-white);
-@text-color-muted: contrast(@background-color, fade(@color-black, 60%), fade(@color-white, 50%));
+@background-color: red;
 ```
+Switch to the web browser. Select your skin from the *Skin menu*. It should show a fiery red editor
 
-### Preview the skin
+![**TinyMCE skin SDK for Silver theme**]({{site.baseurl}}/images/SDKforsilverCustomExample.png)
 
-1. To preview the new skin, run the following command from a terminal:
-```js
-npm run start
-```
-2. Navigate to the address shown in the terminal, usually `localhost:3000`. The TinyMCE skin SDK for Silver theme development environment page will appear in the browser.
-3. In the **Editor examples** section, select the new skin from the **UI skin** drop-down menu.
-![**UI skin**]({{site.baseurl}}/images/default-copy.png)
+This is how you skin TinyMCE: copy variables from the files in the theme folder and paste them into your skin file. There are variables for most things, like spacing between toolbar buttons to letter spacing. Simple yet powerful.
 
-#### Result
+> Tip: You can change the TinyMCE config in `modules/oxide/src/demo/index.html` to suit your particular use case.
 
-TinyMCE editor with all the configured skin changes will appear in the browser screen.
+To style interface elements in the content, such as selection color, drag handles, table of contents, bookmarks etc, copy variables to `modules/oxide/src/less/skins/ui/<skin-name>/content.less` or `content.inline.less` (if inline mode is being used).
 
-### Creating a content CSS file
+## Creating a content CSS file
```

Comment:
This section has the same information as the paragraph above, on line 107? Could we combine line 107 into this section?

Reply:
I removed that, as I believe it was left there by mistake.